.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,20 @@
+# important: assumes that each agent runs a single job at a time
+
+# ignore errors
+set +e
+
+if [[ $BUILDKITE_AGENT_META_DATA_QUEUE == "pipeline-uploader" ]]; then
+  echo "Skipping local pre-exit on `pipeline-uploader` agents"
+  exit 0
+fi
+
+echo "Cleaning up unused docker containers"
+docker ps -a
+for id in $(docker ps -qa); do
+  echo "Killing $id"
+  docker stop -t 1 "$id" || true
+done
+
+docker container prune --force
```

Comment:
`docker system prune --force --volumes --all` will cover the first two lines here. I don't know if it'll clear out networks through 🤔

Reply:
based on docs, it seems like it does